projectroles/tests/test_templatetags.py

```diff
@@ -725,3 +725,184 @@ def test_get_sidebar_links_timeline_view(self):
                     self.assertEqual(app['active'], True)
                 else:
                     self.assertEqual(app['active'], False)
+
+    def test_get_user_links_home(self):
+        """Test get_user_links() on the home view"""
+        url = reverse('home')
+        with self.login(self.user):
+            request = self.req_factory.get(url)
+            request.resolver_match = resolve(url)
+            request.user = self.user
+            self.assertEqual(
+                tags.get_user_links(request),
+                [
+                    {
+                        'name': 'appalerts',
+                        'url': '/alerts/app/list',
+                        'label': 'App Alerts',
+                        'icon': 'mdi:alert-octagram',
+                        'active': False,
+                    },
+                    {
+                        'name': 'example_site_app',
+                        'url': '/examples/site/example',
+                        'label': 'Example Site App',
+                        'icon': 'mdi:rocket-launch-outline',
+                        'active': False,
+                    },
+                    {
+                        'name': 'timeline_site',
+                        'url': '/timeline/site',
+                        'label': 'Site-Wide Events',
+                        'icon': 'mdi:clock-time-eight-outline',
+                        'active': False,
+                    },
+                    {
+                        'name': 'tokens',
+                        'url': '/tokens/',
+                        'label': 'API Tokens',
+                        'icon': 'mdi:key-chain-variant',
+                        'active': False,
+                    },
+                    {
+                        'name': 'userprofile',
+                        'url': '/user/profile',
+                        'label': 'User Profile',
+                        'icon': 'mdi:account-details',
+                        'active': False,
+                    },
+                    {
+                        'name': 'sign-out',
+                        'url': '/logout/',
+                        'label': 'Logout',
+                        'icon': 'mdi:logout-variant',
+                        'active': False,
+                    },
+                ],
+            )
+
+    def test_get_superuser_links_home(self):
+        """Test get_user_links() on the home view as superuser"""
+        url = reverse('home')
+        superuser = self.make_user('superuser')
```

Comment:
We can create the superuser in `setUp()`, to save us from several lines of re-init code in the test cases.